pkg/tsdb/azuremonitor/resourcegraph/azure-resource-graph-datasource.go

```diff
@@ -139,15 +142,15 @@ func (e *AzureResourceGraphDatasource) executeQuery(ctx context.Context, logger
 		dataResponse.Frames = frames
 		return dataResponse
 	}
-
-	model, err := simplejson.NewJson(query.JSON)
+	var model reqAzureBodySubscriptions
```

Comment:
The `dataquery.AzureMonitorQuery` type can be used here instead of defining a new type.

Reply:
Will fix. Thank you!